lte/gateway/c/core/oai/tasks/amf/nas5g_message.cpp

```diff
@@ -61,7 +61,7 @@ int _nas5g_message_plain_encode(unsigned char* buffer,
 
 static int _nas5g_message_protected_encode(
     unsigned char* buffer, const nas_message_security_protected_t* msg,
-    uint32_t length, void* security);
+    uint32_t length, amf_security_context_t* security);
```

Comment:
No change in indentation has been made, only the adjustment in the type to avoid unnecessary cast and increase safeness of code.
The indentation is automatically placed in the devcontainer environment.